algorithms/Python/sorting/count-sort.py

```diff
@@ -0,0 +1,32 @@
+#Code in python for count sort 
+def count_sort(a):
+    max_element = int(max(a))
+    min_element = int(min(a))
+    range_of_elements = max_element - min_element + 1
+    # Create a count array to store count of individual
+    # elements and initialize count array as 0
+    c= [0 for _ in range(range_of_elements)]
+    o= [0 for _ in range(len(a))]
+    for i in range(0, len(a)):
+        c[a[i]-min_element] += 1
+    for i in range(1, len(c)):
+        c[i] += c[i-1]
+    for i in range(len(a)-1, -1, -1):
+        o[c[a[i] - min_element] - 1] = a[i]
+        c[a[i] - min_element] -= 1
+    for i in range(0, len(a)):
+        a[i] = o[i]
+ 
+    return a
+n=int(input("Enter the size of list"))
```

Comment:
It is recommended to add sample test inputs instead of asking the user for the same. 

The repository uses pytest to test the condition of the program, and as it have to capture the program's output , any attempt to take input from the user would cause it to fail.